### DISCUSSION TITLE
Translation in JsonResource breaking Content-Type

### DISCUSSION THREAD

**Discussion by k0mar12** [laravel/framework#42174] [Q&A]
Hey there.

I have JsonResource: 

```
class CountryResource extends JsonResource
{
    /**
     * Transform the resource into an array.
     *
     * @param \Illuminate\Http\Request
     * @return array
     */
    public function toArray($request) : array
    {
        return [
            'slug' => $this->slug,
        ];
    }
}
```

And it works fine, in response im getting **Content-Type:application/json**, great.

But if i will try to add translation, for example: 

```
class CountryResource extends JsonResource
{
    /**
     * Transform the resource into an array.
     *
     * @param \Illuminate\Http\Request
     * @return array
     */
    public function toArray($request) : array
    {
        return [
            'slug' => $this->slug,
            'title' => __('structure::country.' . $this->slug),
        ];
    }
}
```

And at this moment **Content-Type** contain **text/html; charset=UTF-8**

What is the problem, i cant use translation in JsonResource for correct **Content-Type** ?

**Comment by rschoonheim** [comment]
Hello Vladyslav, 

I’m unable to reproduce the described problem. I’ve made a pull request containing code I wrote while trying to reproduce the problem. You can find the pull request [here](https://github.com/rschoonheim/laravel-examples/pull/1/files).

Could you provide us some additional context? What Laravel version are you running? Are there any packages that interfere with Laravels resources?
